sdk/storage/inc/datalake/protocol/datalake_rest_client.hpp

```diff
@@ -238,14 +245,14 @@ namespace Azure { namespace Storage { namespace DataLake {
   {
     std::string Name;
     std::string LastModified;
-    std::string ETag;
+    std::string Etag;
```

Comment:
Why changed?

Reply:
When parsing the response body, server end tag is `etag` not `ETag`.
Also, server is returning `lastModified` so it's camel case and making `etag` one word.